packages/vuetify/src/components/VDatePicker/VDatePicker.ts

```diff
@@ -122,21 +121,24 @@ export default mixins(
           return this.pickerDate
         }
 
-        const date = (this.multiple ? (this.value as string[])[(this.value as string[]).length - 1] : this.value) ||
+        const date = (this.multiple || this.range ? (this.value as string[])[(this.value as string[]).length - 1] : this.value) ||
           `${now.getFullYear()}-${now.getMonth() + 1}`
         return sanitizeDateString(date as string, this.type === 'date' ? 'month' : 'year')
       })(),
     }
   },
 
   computed: {
+    internalMultiple (): boolean {
```

Comment:
probably `isMultiple` will be better

Reply:
Should I put a comment there to explain this compute, like:
// Range selection automatically indicate multiple

Reply:
no, just rename.